frontend/src/constants/links.js

```diff
@@ -6,4 +6,5 @@ export class SITE_PAGES {
   static WISH_GRANTING = "/wish-granting";
   static PROFILE = "/profile";
   static PEOPLE = "/people";
+  static MANAGE = "/manage";
```

Comment:
I meant for the "people" page to be the manage page. Manage probably makes more sense, but we can get rid of the people components at this point if you've replaced them